index.html

```diff
@@ -68,25 +75,471 @@ <h3 class="main-subheading">
+        <img src="#" alt="#" class="join-us-img" />
+      </figure>
+      <h2 class="join-us-heading">Join Our Digital Creative Revolution</h2>
+      <p class="join-us-text">
+        Share your talent with the world and become a part of our ever-growing
+        community of creators.
+      </p>
+      <a href="#" class="join-us-btn">Become A Creator</a>
+    </section>
+    <section class="blog-container">
+      <div class="blog-header">
+        <p class="blog-text">BLOG</p>
+        <h2 class="blog-heading">Latest Expert Design Insights</h2>
+        <button class="blog-cycle-back">&lt;</button>
+        <button class="blog-cycle-back">&gt;</button>
+      </div>
+      <ul class="blogs-list">
+        <li class="blog-item">
+          <article class="article-container">
+            <figure class="article-illustration">
+              <img src="#" alt="#" class="article-img" />
+              <p class="article-tag">TYPOGRAPHY</p>
+            </figure>
+            <h3 class="article-heading">
+              Mastering the Art of Typography: Tips for Eye-Catching Designs
+            </h3>
+            <p class="article-date">5 APRIL 2024</p>
+          </article>
+        </li>
+        <li class="blog-item">
+          <article class="article-container">
+            <figure class="article-illustration">
+              <img src="#" alt="#" class="article-img" />
+              <p class="article-tag">UI/UX DESIGN</p>
+            </figure>
+            <h3 class="article-heading">
+              The Ultimate Guide to UI/UX: Enhancing App User Experience with
+              Design
+            </h3>
+            <p class="article-date">16 MARCH 2024</p>
+          </article>
+        </li>
+        <li class="blog-item">
+          <article class="article-container">
+            <figure class="article-illustration">
+              <img src="#" alt="#" class="article-img" />
+              <p class="article-tag">DESIGN TRENDS</p>
+            </figure>
+            <h3 class="article-heading">
+              2024’s Top Graphic Design Trends: What Would Be Shaping the Future
+              of Design?
+            </h3>
+            <p class="article-date">27 FEBRUARY 2024</p>
+          </article>
+        </li>
+      </ul>
+    </section>
+    <footer class="page-footer">
+      <div class="newsletter-container">
+        <h2 class="footer-heading">Stay ahead of the curve with us.</h2>
+        <form action="#" class="footer-form">
+          <input
+            type="text"
+            placeholder="Enter your email address ..."
+            class="main-search-input"
+            required
+          />
+          <input type="submit" value="Search Now" class="main-search-btn" />
+        </form>
+      </div>
+      <div class="footer-nav-container">
+        <nav class="footer-nav">
+          <ul class="company-nav">
+            <li class="company-item">
+              <h3 class="company-heading">Company</h3>
```

Comment:
A list is an enumeration of homogeneous elements with the same meaning and styling. This means that any number of items can be added to the list, and they should all look the same. 
A heading stands out from this enumeration and is not part of the list.